CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+## [4.0.0] - unreleased
 
+- Rename package to `mongodb/laravel-eloquent`
```

Comment:
```suggestion
- Rename package to `mongodb/laravel`
```

Hello,

Sorry to ask but wouldn't it be better to rename the package to `mongodb/laravel` as it's easy to remember and to type as well...

Thanks!

Reply:
Indeed that's a good question that we are discussing with @alcaeus.

Reply:
We will follow @chalasr [recommendation](https://twitter.com/chalas_r/status/1696099974147096910) and name the composer package like the GitHub project: `mongodb/laravel-mongodb` to eliminate any ambiguity.

Reply:
Hello,

This doesn’t make sense. If you use namespace `MongoDB/Laravel` then the package name should follow it?

Maybe change the namespace as well or should each and every package named twice? Like `mysql/laravel-mysql` it’s already under MySQL organization and you can understand that it’s specially for Laravel. I’m sure you’ll ask what was the reason behind it, well, there is no logical reason behind it.

Anyways, looks like I’m not a collaborator anymore and my suggestion doesn’t have any power. No problem.

Thanks!

Reply:
Hello @divine. 

At first sight I wasn't convinced by this duplication either, but that's the best name we can find.

In `mongodb/laravel-mongodb`, the 1st `mongodb` it the vendor: the MongoDB company taking over the maintenance. The 2nd `laravel-mongodb` is the package name that indicate this is a package for both Laravel (the framework) and MongoDB (the product). 
In the same way, the project needs to be in MongoDB's GitHub organisation. We keep the project name `laravel-mongodb` that is commonly used to refer to this project in issues and docs. When forking the project for contribution, the fork name stay descriptive even if it loose it's org part. 
If we call it `mongodb/laravel`, this can be interpreted as a *fork* of `laravel/laravel`.
Having a match between the project name on GitHub and the package name in Composer make things easier for developers to locate the source code of the package from its name.

Reply:
> Anyway, looks like I’m not a collaborator anymore and my suggestion doesn’t have any power. No problem.

Your work on this project is remarkable. I want you to be able to continue to contribute as you are doing.

Permissions management has changed with the move to MongoDB's GitHub organization.